src/Entity/Storage/AppStorage.php

```diff
@@ -94,6 +94,27 @@ public function loadUnchanged($id) {
     return parent::loadUnchanged($id);
   }
 
+  /**
+   * Load unchanged for when $id is an App id.
```

Comment:
I would rephase this: "Load app by UUID.".

Reply:
done